### PR TITLE
Devpages restructure

### PR DESCRIPTION
**Pull request by Markusdreyer**
## Recap:
* Change headers levels to comply with devpages (Single h1)
* Add postman guide
* Move Postman and Test environment info from README.md out to separate files

### CODE REVIEW

File: contact.md [hunk 1]
```diff
@@ -19,27 +20,27 @@ Version: 2.0.1.
...
+| Using Vipps through a partner                                                                                                   | Please [contact your partner directly](https://www.vipps.no/produkter-og-tjenester/bedrift/ta-betalt-paa-nett/ta-betalt-paa-nett/#kom-i-gang-med-vipps-pa-nett-category-3) |
+| Ordering Vipps through a partner                                                                                                | Please contact partnerbestilling@vipps.no                                                                                                                                  |
+| If you are a partner and need help                                                                                              | Please contact partnerbestilling@vipps.no                                                                                                                                  |
```
Comment (tht13) [inline contact.md:30]: So much white space man
https://guides.github.com/features/mastering-markdown/#GitHub-flavored-markdown
look at the tables section

Comment (Markusdreyer) [reply contact.md:30]: Hmm.. Seems like my VSCode formatter is having a feast. I'll look into it

File: vipps-getting-started.md [hunk 2]
```diff
@@ -45,26 +76,25 @@ Please see the general developer information: https://github.com/vippsas/vipps-d
 
 # Getting a test account (and a production account)
 
+>>>>>>> a89023ee6b2d4a88657165a03423256ecbf7c20f
```
Comment (niklud) [inline vipps-getting-started.md:79]: Seems like merge conflict have been included in commit.

Comment (Markusdreyer) [reply vipps-getting-started.md]: fixed in bab9c39

File: contact.md (hunk 1 above)
Comment (Markusdreyer) [reply contact.md:30]: fixed in bab9c39

Comment (niklud) [reply contact.md:30]: You seem to have fixed it in readme.md, but not here?

Comment (Markusdreyer) [reply contact.md]: 🤦‍♂️Fixed in 61d029a

**Pull request merged**